src/mobot_client/mobot.py

```diff
@@ -74,277 +82,254 @@ def __init__(self):
-            try:
-                active_drop_session = DropSession.objects.get(
-                    customer=customer,
-                    drop__drop_type=DropType.AIRDROP,
-                    state__gte=SessionState.READY_TO_RECEIVE_INITIAL,
-                    state__lt=SessionState.COMPLETED,
-                )
-            except (Exception,):
-                #  there is *not* an active airdop session; continue exploring what to do
-                pass
-            else:
-                # there *is* an active airdrop session.
-                print(f"found active drop session in state {active_drop_session.state}")
-                # manual_override means that a human is monitoring the chat via a linked
-                # device and wants to respond rather than having MOBot respond, so do
-                # nothing.
-                if active_drop_session.manual_override:
-                    return
-
-                # Dispatch to the active_airdrop session handler
-                air_drop = AirDropSession(self.store, self.payments, self.messenger)
-                air_drop.handle_active_airdrop_drop_session(
-                    message, active_drop_session
-                )
-                return
+        self.messenger.log_and_send_message(
+            customer, message.source, ChatStrings.SUBSCRIBE_NOTIFICATIONS
+        )
 
-            # see if there's an active item drop session
-            try:
-                active_drop_session = DropSession.objects.get(
-                    customer=customer,
-                    drop__drop_type=DropType.ITEM,
-                    state__gte=ItemSessionState.WAITING_FOR_PAYMENT,
-                    state__lt=ItemSessionState.COMPLETED,
-                )
-            except (Exception,):
-                #  there is not an active item dop sesssion; continue to exploring what to do
-                pass
+    def handle_new_drop_session(self, customer: Customer, message, active_drop: Drop):
+        if active_drop.drop_type == DropType.AIRDROP:
+        # if this is an airdrop session, dispatch to the
+        # no_active_airdrop session handler to initiate a session
+            air_drop = AirDropSession(self.store, self.payments, self.messenger)
+            air_drop.handle_no_active_airdrop_drop_session(
+                customer, message, active_drop
+            )
+        elif active_drop.drop_type == DropType.ITEM:
+            # if this is an item drop session, dispacth to the
+            # no_active_item_drop session handler to initiate a session
+            item_drop = ItemDropSession(self.store, self.payments, self.messenger)
+            item_drop.handle_no_active_item_drop_session(
+                customer, message, active_drop
+            )
+        else:
+            raise Exception("Unknown Drop Type")
+
+    def default_handler(self, message, match):
+        # Store the message
+        self.logger.info(f"\033[1;33m NOW ROUTING CHAT\033[0m {message}")
```

Comment:
I don't think we want these ANSI colors in our logs, especially once we start logging into logit.io or something like that.